### PR TITLE
Fixes to Annual Reviews author-date

### PR DESCRIPTION
**Pull request by bwiernik**
1. Fix display of paper-conference (previously it had not displayed any container information)
2. Add support for status and original date variables

### CODE REVIEW

File: annual-reviews-author-date.csl [hunk 1]
```diff
@@ -224,16 +260,43 @@
...
+            <text macro="locator"/>            
+          </if>
+          <else>
+            <text macro="container-title"/>
+            <text macro="locator"/>
+            <choose>
+              <if variable="publisher" match="any">
+                <group delimiter=". " prefix=". ">
+                  <text macro="publisher"/>
+                  <text macro="status"/>
```
Comment (adam3smith) [inline annual-reviews-author-date.csl]: this looks like we could get status twice -- once from the locator macro, one here. Am I missing something?

**Comment by adam3smith** [comment]
so, what do you see as being in the "status" variable? And does it really never replace the date, e.g. for "in press" when no date of publication is assigned?

File: annual-reviews-author-date.csl (hunk 1 above)
Comment (bwiernik) [inline annual-reviews-author-date.csl]: There is a conditional to test for 'volume' in both the locator macro and the container macro that controls the placement of status. The point is to accommodate the different delimiters needed for journals/journal-like proceedings and books/book-like proceedings.

**Comment by bwiernik** [comment]
For Annual Reviews, "status" should always be "In press". It is placed at the end of the reference, after as much of the rest of the reference information as possible. It never replaces the date (some date, usually the expected publication date, should always be included). It only appears in the bibliography, not in the citations. 

Pages 19 and 29 here:
http://www.annualreviews.org/userimages/ContentEditor/1343842689197/AuthorHandbook-GREEN.pdf

**Comment by adam3smith** [comment]
OK, sounds good

**Pull request merged**